requirements.txt

```
@@ -3,7 +3,10 @@ dipy
 # https://github.com/tensorflow/tensorflow/issues/44467
 h5py~=2.10.0
 Keras==2.3.1
-ivadomed
+# PyTorch's Linux distribution is very large due to its GPU support,
+# but we only need that for training models. Our users only need CPU.
+--extra-index-url https://download.pytorch.org/whl/cpu
+ivadomed@git+https://github.com/ivadomed/ivadomed@ng/cu113
```

Remember to revert back to

```suggestion
ivadomed
```

once ivadomed publishes this fix.

---

Release is out, I've put this back how it should be.

I've added a minimum version now, to make sure anyone upgrading overtop is sure to get the working version of ivadomed. Not that that could possibly be an issue until #1526. But it's good to follow the conventions.